pages/common/odps-ddl.md

```diff
@@ -0,0 +1,23 @@
+# odps ddl
+
+> Run ddl task in ODPS (Open Data Processing Service).
```

Comment:
@waldyrious 

Em. DDL a.k.a data definition language...used a lot in database area.

Since all those commands deal with tables, I have changed the name to odps-table